### PR TITLE
Model db schema

### PR DESCRIPTION
**Pull request by mariuscoto**
This is the implementation of #39.

### CODE REVIEW

**Comment by mariuscoto** [comment]
ok, @Aleks23, it's now your turn. look over these commits and what I want you to do is the same thing with the editions
* build a view that lists all editions and displays a form to add new editions (in the form provide activity_id and edition name)
* add a route that precesses the form and adds the edition to the corresponding activity

File: config/models/activity.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var mongoose = require('mongoose')
+
+
+var activitySchema = mongoose.Schema({
+    name            : String,
+    description     : String,
+    edition         : {type: [Edition], default: []}
+})
+
+var Edition = mongoose.Schema({
```
Comment (maria) [inline config/models/activity.js]: `editionSchema`.. consistency++

File: config/models/activity.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    name            : String,
+    people 			: [String]
```
Comment (maria) [inline config/models/activity.js]: shouldn't it have a `period` field of type `datetime`?

File: config/models/event.js [hunk 3]
```diff
@@ -12,8 +11,12 @@ var eventSchema = mongoose.Schema({
     description  : String,
     publisher    : String,
     membersOnly  : {type: Boolean, default: false},
-    tags         : [String]
-});
+    tags         : [String],
+
+    activity_id  : String,
```
Comment (maria) [inline config/models/event.js]: `activityId` I think is better, we also have `ObjectId` etc.

File: config/models/user.js [hunk 4]
```diff
@@ -1,27 +1,23 @@
...
         avatar 		 : String,
     }
-});
+})
 
-// generating a hash
-userSchema.methods.generateHash = function(password) {
-    return bcrypt.hashSync(password, bcrypt.genSaltSync(8), null);
-};
+var AEdR = mongoose.Schema({
```
Comment (maria) [inline config/models/user.js]: `AEdR` comes from?

File: config/models/activity.js (hunk 2 above)
Comment (maria) [inline config/models/activity.js]: This way we can display editions in a timeline. :)

File: routes/activities.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+exports.add = function(req, res) {
+  var new_activity = {
+    name:      req.body.name,
+    description: req.body.description
+  }
+
+  new Activity(new_activity).save()
+  res.redirect('/activities')
+}
```
Comment (maria) [inline routes/activities.js]: Empty line at the end of file :octocat:

**Pull request merged**